book.md

```diff
@@ -2110,16 +2112,101 @@ architectural state. [Could we find a good reference that explains
 micro-architectural versus architectural state in more detail? Is "Computer
 Architecture: A Quantitative Approach" the best reference available?]{.todo}
 
+**Faulting instructions**\index{faulting instructions} are instructions that
+generate an exception at run-time. Many instructions can generate an exception,
+and are hence **potentially faulting**. For example most load and store
+instructions generate an exception when the accessed address is not mapped.
+Since so many instructions can generate an exception, processors typically
+speculate that they do not generate an exception to enable more parallel
+execution.
+
+When an instruction faults, the execution typically continues at another
+location. Any instructions later in the instruction stream which are
+speculatively executed before the fault is detected are also called **transient
+instructions**\index{transient instructions}.
+
+There is a kind of control dependency between every potentially-faulting
+instruction and the next one, as the next instruction to be executed depends on
+whether the instruction generates an exception or not. We call out this
+dependency separately here as the transient execution attacks we'll describe
+next get classified based on whether they make use of transient instructions
+after a misprediction, or transient instructions after a faulting instruction.
+
+### Transient Execution Attacks
+
 **Transient execution attacks**\index{transient execution attacks} are a
 category of side-channel attacks that use the micro-architectural side-effects
 of transient execution as a side channel.
 
+The publication of the Spectre\index{Spectre} [@Kocher2019] and
+Meltdown\index{Meltdown} [@Lipp2018] attacks in 2018 started a period in which a
+large number of transient attacks were discovered and published. Most of them
+were given specific names, such as ZombieLoad, NetSpectre, LVI, Straight-line
+Speculation, etc. New variants continue to be published regularly.
+
+Covering each one of them in detail here would make the book overly lengthy, and
+may not necessarily help much with gaining a better insight in the common
+characteristics of transient attacks. Therefore, we'll try to put them into a
+few categories and describe the characteristics of each category.
+
+::: TODO
```

Comment:
If you feel like, references from https://github.com/MattPD/cpplinks/blob/master/comparch.micro.channels.md may be useful (both the General section, https://github.com/MattPD/cpplinks/blob/master/comparch.micro.channels.md#general, as well as https://github.com/MattPD/cpplinks/blob/master/comparch.micro.channels.md#speculation).